doc/api/cli.md

```diff
@@ -623,8 +623,8 @@ added: v6.9.0
 -->
 
 Load an OpenSSL configuration file on startup. Among other uses, this can be
-used to enable FIPS-compliant crypto if Node.js is built with
-`./configure --openssl-fips`.
+used to enable FIPS-compliant crypto if Node.js is built
```

Comment:
Will `./configure --openssl-fips` still be accepted. Want to understand if this could break existing workflows/builds

Reply:
Yeah, it will still be accepted at configuration time to specify the directory where the FIPS fipscanister.o is located.

Reply:
@mhdawson The [ubi81_sharedlibs_openssl111fips_x64](https://ci.nodejs.org/job/node-test-commit-linux-containered/nodes=ubi81_sharedlibs_openssl111fips_x64/) CI job runs with `configure --openssl-is-fips` (we didn't update the job when https://github.com/nodejs/node/pull/36341 landed).

Reply:
Thanks for the quick confirmation.

Reply:
ah sorry confused `--open-is-fips` with `--openssl-fips` configure options.

Reply:
I don't think `--openssl-fips` works before on Node.js 14 based on `configure.py`?
https://github.com/nodejs/node/blob/95b9240a266c46b732e9d15bc9ab2b31f12a871d/configure.py#L1382-L1383

(`--openssl-is-fips` does work and is tested in the CI.)

Reply:
If I recall correctly it is not possible to enable FIPS with the OpenSSL version that is statically linked with Node.js. This is still the case with [upstream/master](https://github.com/nodejs/node/blob/481c1609d8965b0c93d56180b0efbc7e06b68b0c/configure.py#L1471-L1472). 
But it is possible to dynamically link to an OpenSSL version that does support FIPS (which is the use case at Red Hat).

Reply:
@danbev that is my understanding as well.